data_structures/binary_search_tree.cpp

```diff
@@ -98,6 +98,10 @@ void BFT(node *n) {
     }
 }
 
+/*
+    Prints the preorder traversal starting from the node n
```

Comment:
please follow doxygen documentation for documenting functions.
https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/CONTRIBUTING.md